test/setup.ts

```diff
@@ -36,11 +37,20 @@ use(sinonChai);
 use(chaiSorted);
 use(deepEqualInAnyOrder);
 
+config();
 process.env.NODE_ENV = 'test';
-process.env.HTTP_PORT = '3001';
-process.env.TYPEORM_CONNECTION = 'sqlite';
-process.env.TYPEORM_DATABASE = ':memory:';
-process.env.TYPEORM_SYNCHRONIZE = 'true';
+if (!process.env.TYPEORM_CONNECTION || (process.env.TYPEORM_CONNECTION === 'sqlite' && !(process.env.SKIP_SQLITE_DEFAULTS === 'true'))) {
+  console.log('Setting sqlite defaults');
+  process.env.HTTP_PORT = '3001';
+  process.env.TYPEORM_CONNECTION = 'sqlite';
+  process.env.TYPEORM_DATABASE = ':memory:';
+  process.env.TYPEORM_SYNCHRONIZE = 'true';
+}
+
+// Always sync in sqlite.
+if (process.env.TYPEORM_CONNECTION === 'sqlite') {
+  process.env.TYPEORM_SYNCHRONIZE = 'true';
```

Comment:
If we do not wish to override SQLite defaults, why do we then still override this particular variable? I can assume there are use cases when you want to use a local database file, but do not want it to synchronize (for example when the synchronization is doing something wrong?).